scania-digital-design-system/tegel
# test(breadcrumbs): adding unit tests

**Describe pull-request**  
initial tests for breadcrumbs

**Solving issue**  
Fixes: [CDEP-3009](https://tegel.atlassian.net/browse/CDEP-3009)

**How to test**  
1. Checkout code
2. Make sure that each test runs correctly via `npm run test`


[CDEP-3009]: https://tegel.atlassian.net/browse/CDEP-3009?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
